pkg/apis/mattermost/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,5 +1,21 @@
 // +build !ignore_autogenerated
 
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
 // Code generated by deepcopy-gen. DO NOT EDIT.
```

Comment:
Could we configure a .gitattributes file to mark these as `linguist-generated` and suppress the diff by default? (https://thoughtbot.com/blog/github-diff-supression)

Reply:
You are right! That is much cleaner to review now.